crates/swc_ecma_minifier/src/compress/optimize/conditionals.rs

```diff
@@ -616,6 +616,81 @@ impl Optimizer<'_> {
                 }))
             }
 
+            (Expr::Seq(left), Expr::Seq(right)) => {
+                let last = left.exprs.len().min(right.exprs.len()) - 1;
+
+                let mut idx = 0;
+
+                while idx <= last
+                    && left.exprs[last - idx].eq_ignore_span(&right.exprs[last - idx])
+                    && !matches!(&*left.exprs[last - idx], Expr::Yield(..) | Expr::Fn(..))
```

Comment:
I think we need `Expr::Arrow`, too.

Reply:
Why? I just copy this from L356, and I don't believe the original is necessary

Reply:
Ah, I see. I just thought `Expr::Arrow `is required because there's `Expr::Fn`

Reply:
Do you remember why the original restriction is written?